### PR TITLE
Removed all dead links that could not find a replacement for

### PR DESCRIPTION
**Pull request by Choromanski**

### CODE REVIEW

**Comment by jackhumbert** [comment]
Awesome! I think there are some updated links for the Knops stuff - @Pawnerd, would you mind updating these when you get the chance?

**Comment by Pawnerd** [comment]
I will have a look on it. Currently there is a lot going on with the roll out of the new knops tool. Once that is done, it's time to update the docs :).

**Comment by Choromanski** [comment]
Placed the knops links back in, should be good to merge

**Comment by Choromanski** [comment]
There does however seem to be an error compiling knops/mini:knops but that is unrelated to what I have edited

**Comment by Choromanski** [comment]
@jackhumbert fixed up the knops.io links
@Pawnerd looks as if https is not supported on knops.io i don't know if you maintain the website or not just an fyi

File: keyboards/clueboard/66/readme.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # Clueboard 66%
 
-![Clueboard](https://static1.squarespace.com/static/55c13bdee4b099be5dcb82eb/t/5867eeaad2b857fd0d196f4b/1494021396651/IMGP4201.jpg?format=1500w)
-
```
Comment (jackhumbert) [inline keyboards/clueboard/66/readme.md]: @skullydazed would you mind updating these when you get the chance?

**Comment by Pawnerd** [comment]
@Choromanski https is indeed not supported at this moment. Currently I can't access the server files. When I have time I will move the website to a new server with letsencrypt https :).

**Comment by drashna** [comment]
There are a number of merge conflicts on this now.  Would you mind cleaning this up?

**Comment by Choromanski** [comment]
@drashna Unfortunately I had deleted this branch. Opened a new pull request #5503 @Pawnerd @jackhumbert

**Pull request closed without merging**